test/extension/loadable_extension_optimizer_demo.cpp

```diff
@@ -142,6 +143,8 @@ class WaggleExtension : public OptimizerExtension {
 		WriteChecked(sockfd, &len, sizeof(idx_t));
 		// close the socket
 		close(sockfd);
+
+		return false;
```

Comment:
Should this return true, just to test the rest of the system on the true condition?

Reply:
Not sure. This looks complicated. I can maybe add a new basic optimizer for the coverage.